core/src/test/java/com/graphhopper/routing/RoutingAlgorithmIT.java

```diff
@@ -104,13 +105,14 @@ public void testMonacoMotorcycle()
         list.add(new OneRun(43.730729, 7.42135, 43.727697, 7.419199, 2703, 119));
         list.add(new OneRun(43.727687, 7.418737, 43.74958, 7.436566, 3749, 170));
         list.add(new OneRun(43.728677, 7.41016, 43.739213, 7.4277, 3175, 169));
-        list.add(new OneRun(43.733802, 7.413433, 43.739662, 7.424355, 2423, 141));
-        list.add(new OneRun(43.730949, 7.412338, 43.739643, 7.424542, 2253, 120));
+        list.add(new OneRun(43.733802, 7.413433, 43.739662, 7.424355, 5636, 298));
+        list.add(new OneRun(43.730949, 7.412338, 43.739643, 7.424542, 5467, 277));
```

Comment:
Yes I compared it with shortest. I think the issue is related to speed limites. Probably in the old version of Monaco the speed limits are not set properly. Since I changed the default speed lmits, the routing could be wrong. That would be my guess, since the elevation thing is only relevant when using the curvature routing.